src/web3-wrapper/rpc/evmRPCSender.ts

```diff
@@ -18,7 +21,9 @@ export class EvmRPCSender extends AbstractRPCSender {
   constructor(
     rpcInfos: RpcInfo[],
     private networkId: number | string,
-    private rpcProviderFn: (provider: ethers.providers.StaticJsonRpcProvider) => Promise<any>,
+    private networkName: string,
+    private rpcProviderFn: (provider: ArchiveJsonRpcProvider) => Promise<any>,
+    private proxyServerUrl: string,
```

Comment:
Can we make this `proxyServerUrl` optional parameter?

This is also called by external BP developers and I think there is a case that they don't provide proxyServerUrl